### ISSUE TITLE
🎁 [Feature Request]: upgrade polkadot-sdk to `v1.13.0` 

### ISSUE THREAD

**Issue by 4meta5** [OpenZeppelin/polkadot-runtime-templates#205]
### templates

- [X] Generic Runtime Template
- [X] Evm Template

### What is the feature you would like to see?

1. choose polkadot-sdk version to upgrade before/until higher priority issues are done. `v1.12.0` was released 5 days ago as of writing this
2. upgrade all forks required for external dependencies 
3. document and iteratively automate all processes

(1) requires beforehand agreement as a group to choose the stable version. The work required for (2) will determine timing and prioritization of (1).

It may not be ideal to be the first to upgrade all the dependencies i.e. frontier as well as any additional external dep forks required to finish #189 i.e. ORML, Acala, Moonbeam, HydraDX

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines

**Comment by ozgunozerk** [comment]
We have a specific issue for the 3rd point made for this issue. Better to dump all info we gathered on update process there #93

**Comment by ggonzalez94** [comment]
@4meta5 moving this to Milestone 3 since you are already working on it

**Comment by ozgunozerk** [comment]
the other issue is not a replacement. No need to close this one. The PR addressing the update will close both of them. 
We previously planned to upgrade to `v1.13.0` and every single one of us contributed to this task. 

After that, Nikita suggested we upgrade to even newer version, and I opened a new issue for that, which doesn't neglect this one

**Comment by ozgunozerk** [comment]
Assigned also @KitHat since he contributed to evm-node part and fuzzer

**Comment by ozgunozerk** [comment]
Yes and all the upgrades build off of the work done in the first time we upgraded, so I should assign myself to every upgrade? Doesn't make sense to me, every issue is scoped for its own sake afaik.

I didn't clear anyone, nor denied anybody's work. On the contrary, I'm thanking to everyone at every possible situation. 

I didn't close this issue, you did. I just chunked a big task into 2.

I don't get the reason to even discuss this, not productive at all and also hurting the team spirit in my opinion. GitHub comments are certainly not the place for these kinds of discussions.
